.devcontainer/db/init-db.sh

```diff
@@ -19,7 +19,11 @@ echo "Configuring md5 authentication in $PGDATA/pg_hba.conf"
 echo 'local all all trust' > $PGDATA/pg_hba.conf
 echo "host all all all md5" >> $PGDATA/pg_hba.conf
 
-# Standard test account for Npgsql
-psql -U postgres -c "CREATE USER npgsql_tests SUPERUSER PASSWORD 'npgsql_tests'"
-psql -U postgres -c "CREATE DATABASE npgsql_tests OWNER npgsql_tests"
-psql -U postgres -c "CREATE EXTENSION ltree" npgsql_tests
+# Standard test account for Npgsql and enable extensions
+psql -U postgres <<EOF
+    CREATE USER npgsql_tests SUPERUSER PASSWORD 'npgsql_tests';
+    CREATE DATABASE npgsql_tests OWNER npgsql_tests;
+    CREATE EXTENSION ltree npgsql_tests;
+    CREATE EXTENSION IF NOT EXISTS plpgsql npsql_tests;
```

Comment:
Is there an actual case where plpgsql isn't installed in PG?

Reply:
good point. i saw it in a document (that i can no longer find) for installing postgis. i'll remove it